packages/lib-utils/src/components/table-view/FilterChips.tsx

```diff
@@ -0,0 +1,53 @@
+import { Chip, ChipGroup, Button } from '@patternfly/react-core';
+import * as React from 'react';
+import './filter-chips.scss';
+
+export type FilterChipItem = {
+  /* Label of a parameter used for filtering. */
+  label: string;
+  /* Column name for given filtering parameter. */
+  id: string;
+};
+
+export interface FilterChipsProps {
+  filters?: FilterChipItem[];
+  filterValues?: Record<string, string>;
+  onDelete?: (key?: string) => void;
+}
+
+const FilterChips: React.FunctionComponent<FilterChipsProps> = ({
+  filters = [],
+  filterValues = {},
+  onDelete = () => undefined,
+}) => {
+  const groupedFilters = Object.keys(filterValues).map((key) => (
+    <ChipGroup
+      className="dps-table-view-chips"
+      key={`group-${key}`}
+      categoryName={filters.find((item) => item.id === key)?.label}
+    >
+      <Chip
+        key={filterValues[key]}
+        onClick={(event) => {
+          event.stopPropagation();
```

Comment:
@fhlavac do we also need to use [`preventDefault` ](https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault)? 

We could create a common click handler for this to use in other places in a utility.